powershell/public/Get-MtGraphScope.ps1

```diff
@@ -28,6 +33,9 @@ Function Get-MtGraphScope {
         # If specified, the cmdlet will include the scope to send email (Mail.Send).
         [Parameter(Mandatory = $false)]
         [switch] $SendMail,
+        # If specified, the cmdlet will include the scope to send Teams Channel Messages (ChannelMessage.Send).
+        [Parameter(Mandatory = $false)]
+        [switch] $SendChannelMessage,
```

Comment:
Let's call this SendTeamsMessage to make it clear. Not many folks would understand Channel.